pkg/controller/daemon/daemon_controller.go

```diff
@@ -197,13 +220,13 @@ func NewDaemonSetsController(
 	return dsc, nil
 }
 
-func (dsc *DaemonSetsController) addDaemonset(obj interface{}) {
+func (dsc *DaemonSetsController) addDaemonset(logger klog.Logger, obj interface{}) {
 	ds := obj.(*apps.DaemonSet)
-	klog.V(4).Infof("Adding daemon set %s", ds.Name)
+	logger.V(4).Info("Adding daemon set", "daemonset", ds.Name)
```

Comment:
```suggestion
	logger.V(4).Info("Adding daemon set", "daemonset", klog.KObj(ds))
```

And the same everywhere else